InetaAdmin/global.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "projects": [ "src", "test" ],
   "sdk": {
-    "version": "1.0.0-rc1-update1"
+    "version": "1.0.0-preview2-003121"
```

Comment:
You need to install the last asp.net core tooling sdk for visual stdio. (here i have used the latest at the time which is preview 2).
download the sdk: https://www.microsoft.com/net/core#windows
and change the sdk version accordingly to your downloaded version.